src/main/java/org/craftercms/deployer/impl/upgrade/operations/RemoveProcessorUpgradeOperation.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (C) 2007-2023 Crafter Software Corporation. All Rights Reserved.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as published by
+ * the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.craftercms.deployer.impl.upgrade.operations;
+
+import org.apache.commons.configuration2.HierarchicalConfiguration;
+import org.craftercms.commons.config.ConfigurationException;
+import org.craftercms.deployer.api.Target;
+
+import java.util.*;
+
+import static org.craftercms.commons.config.ConfigUtils.getRequiredStringProperty;
+import static org.craftercms.deployer.impl.DeploymentConstants.PROCESSOR_NAME_CONFIG_KEY;
+
+/**
+ * Operation to remove a new processor to the pipeline of the target
```

Comment:
"Operation to remove a processor from the pipeline of the target"

Reply:
I updated this. Thank you!